test/metatx/ERC2771Context.test.js

```diff
@@ -22,7 +23,7 @@ contract('ERC2771Context', function (accounts) {
     this.domain = {
       name,
       version,
-      chainId: await web3.eth.getChainId(),
```

Comment:
what is wrong with `web3.eth.getChainId()`, and why do we need an helper to do the rpc call when web3 does it right ?